src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -22,10 +22,23 @@
 #include <ripple/app/main/Application.h>
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/data/DatabaseCon.h>
+#include <ripple/core/ConfigSections.h>
 #include <boost/format.hpp>
+#include <boost/tokenizer.hpp>
 
 namespace ripple {
 
+namespace detail
+{
+/** Amendments that are "compiled into" this app
+
+    Add amendments to this collection at build time to enable them on this
+   server. The first element of the pair is the id, the second element is
+   the "friendly name".
+*/
+std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
```

Comment:
One of the requirements of RIPD-746 was "Read enabled features from static data embedded in the application". This collection supports that. The idea is people can add amendments here by changing the ::ctor from empty to one with an initializer list of id,name pairs.